conf/app.routes

```diff
@@ -20,3 +20,6 @@ POST       /pay-frequency                        controllers.PayFrequencyControl
 
 GET        /pay-method                        controllers.PayMethodController.onPageLoad()
 POST       /pay-method                        controllers.PayMethodController.onSubmit()
+
+GET        /pay-date                  controllers.PayDateController.onPageLoad()
```

Comment:
could you rename this controller to `LastPayDate` and the endpoint to `last-pay-date` - the intent is more clear then

Reply:
sure